### PR TITLE
Remove legacy Icons, use only new icons

### PR DESCRIPTION
**Pull request by peter-sanderson**

### CODE REVIEW

**Review by jvaclavik — APPROVED** [review]

**Review by seibei-iguchi — APPROVED** [review]

File: suite-common/icons/assets/icons/accounts.svg [hunk 1]
```diff
@@ -1,3 +1 @@
...
...9.245,3.402 19.444,3.477 19.642 C 3.625 20.032,3.958 20.365,4.368 20.533 L 4.580 20.620 12.000 20.620 L 19.420 20.620 19.632 20.533 C 20.042 20.365,20.365 20.042,20.533 19.632 L 20.620 19.420 20.620 12.000 L 20.620 4.580 20.533 4.368 C 20.365 3.958,20.032 3.625,19.642 3.477 C 19.444 3.402,19.247 3.400,11.990 3.402 C 4.801 3.404,4.533 3.406,4.348 3.479 M11.240 12.000 L 11.240 19.120 8.060 19.120 L 4.880 19.120 4.880 12.000 L 4.880 4.880 8.060 4.880 L 11.240 4.880 11.240 12.000 M19.120 5.820 L 19.120 6.760 15.940 6.760 L 12.760 6.760 12.760 5.820 L 12.760 4.880 15.940 4.880 L 19.120 4.880 19.120 5.820 M19.120 9.000 L 19.120 9.760 15.940 9.760 L 12.760 9.760 12.760 9.000 L 12.760 8.240 15.940 8.240 L 19.120 8.240 19.120 9.000 M19.120 12.000 L 19.120 12.760 15.940 12.760 L 12.760 12.760 12.760 12.000 L 12.760 11.240 15.940 11.240 L 19.120 11.240 19.120 12.000 M19.120 15.000 L 19.120 15.760 15.940 15.760 L 12.760 15.760 12.760 15.000 L 12.760 14.240 15.940 14.240 L 19.120 14.240 19.120 15.000 M19.120 18.180 L 19.120 19.120 15.940 19.120 L 12.760 19.120 12.760 18.180 L 12.760 17.240 15.940 17.240 L 19.120 17.240 19.120 18.180 " stroke="none" fill="black" fill-rule="evenodd"></path></svg>
```
Comment (vytick) [inline suite-common/icons/assets/icons/accounts.svg]: are those svgs optimised? Each that I checked looks much more complex than before

Comment (vytick) [reply suite-common/icons/assets/icons/accounts.svg]: I tried one randomly in https://svgoptimizer.com/ and it made it 30% smaller

Comment (jvaclavik) [reply suite-common/icons/assets/icons/accounts.svg]: We converted all icons to be implemented without stroke. All major design systems like MUI are using it this way.

There are multiple issues with stroke:

1) you can't change color of the icon properly
2) you can't scale properly

image

**Comment by STew790** [comment]
**QA OK**
I haven't found any legacy icons.

**Info**
24.9.1 e716494

**Pull request merged**